lib/langchain.rb

```diff
@@ -152,6 +152,7 @@ module ActiveRecord
   module OutputParsers
```

Comment:
Any chance that you could start a section in the README and document the Output Parsers, usage, etc.?

Reply:
It'll be a pleasure

Reply:
Added first draft of README section, if you could advise any feedback? cheers.